### DISCUSSION TITLE
Flutter auto upgrade ruins the project EVERY TIME it happens

### DISCUSSION THREAD

**Discussion by Rejvenn** [community/community#37135] [Mobile] [answered]
Every time flutter gets automatically upgraded I can't run my app cause of Xcode errors. Can you help me fix this ? This is like the 5th time this happens and this time I can't get the app to work on iOS Simulator (and/or other devices)

                    ^^^^^^^
    ../../../../Shared/flutter/packages/flutter/lib/src/painting/text_painter.dart:367:24: Error: The method 'dispose' isn't defined for the class 'Paragraph'.
     - 'Paragraph' is from 'dart:ui'.
    Try correcting the name to the name of an existing method, or defining a method named 'dispose'.
          _layoutTemplate?.dispose();
                           ^^^^^^^
    ../../../../Shared/flutter/packages/flutter/lib/src/painting/text_painter.dart:426:22: Error: The method 'dispose' isn't defined for the class 'Paragraph'.
     - 'Paragraph' is from 'dart:ui'.
    Try correcting the name to the name of an existing method, or defining a method named 'dispose'.
        _layoutTemplate?.dispose();
                         ^^^^^^^
    ../../../../Shared/flutter/packages/flutter/lib/src/painting/text_painter.dart:445:22: Error: The method 'dispose' isn't defined for the class 'Paragraph'.
     - 'Paragraph' is from 'dart:ui'.
    Try correcting the name to the name of an existing method, or defining a method named 'dispose'.
        _layoutTemplate?.dispose();
                         ^^^^^^^
    ../../../../Shared/flutter/packages/flutter/lib/src/painting/text_painter.dart:1182:22: Error: The method 'dispose' isn't defined for the class 'Paragraph'.
     - 'Paragraph' is from 'dart:ui'.
    Try correcting the name to the name of an existing method, or defining a method named 'dispose'.
        _layoutTemplate?.dispose();
                         ^^^^^^^
    ../../../../Shared/flutter/packages/flutter/lib/src/painting/text_painter.dart:1184:17: Error: The method 'dispose' isn't defined for the class 'Paragraph'.
     - 'Paragraph' is from 'dart:ui'.
    Try correcting the name to the name of an existing method, or defining a method named 'dispose'.
        _paragraph?.dispose();
                    ^^^^^^^
    ../../../../Shared/flutter/packages/flutter_test/lib/src/matchers.dart:2146:56: Error: Member not found: 'isCheckStateMixed'.
              if (isCheckStateMixed != null) SemanticsFlag.isCheckStateMixed: isCheckStateMixed,
                                                           ^^^^^^^^^^^^^^^^^
    ../../../../Shared/flutter/packages/flutter_test/lib/src/window.dart:731:110: Error: The getter 'nativeSpellCheckServiceDefined' isn't defined for the class 'PlatformDispatcher'.
     - 'PlatformDispatcher' is from 'dart:ui'.
    Try correcting the name to the name of an existing getter, or defining a getter or field named 'nativeSpellCheckServiceDefined'.
      bool get nativeSpellCheckServiceDefined => _nativeSpellCheckServiceDefinedTestValue ?? _platformDispatcher.nativeSpellCheckServiceDefined;
                                                                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    Failed to package /Users/fitwaveit/projekty/fitwavenew/fitwave.
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in dependency order
    /Users/fitwaveit/projekty/fitwavenew/fitwave/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.4.99. (in target 'leveldb-library' from project 'Pods')

    Result bundle written to path:
    	/var/folders/8_/rsbs690x6vjf9j70drfg456h0000gn/T/flutter_tools.OmOTlH/flutter_ios_build_temp_dirOHwQFf/temporary_xcresult_bundle
2

Could not build the application for the simulator.
Error launching application on iPhone 12.

**Comment by Iron-Ham** [comment, marked as answer]
Hi @Rejvenn

This repo is for discussion of the GitHub Mobile app -- I think it may be best to transfer this to [Flutter](https://github.com/flutter/flutter).
